subgraphs/marketplace/src/helpers/model.ts

```diff
@@ -21,6 +21,9 @@ function createCollection(
     collection.listings = [];
     collection.name = name;
     collection.standard = standard;
+    collection.stats = id;
+
+    new Stats(id).save();
```

Comment:
This order of operations may not be working.

I can get the stats by collection ID:
```
{
  stats(id: "0x0af85a5624d24e2c6e7af3c0a0b102a28e36cea3") {
    id
    floorPrice
  }
}
```

But can't get the stats child within the collection:
```
{
  collection(id: "0x0af85a5624d24e2c6e7af3c0a0b102a28e36cea3") {
    id
    stats {
      floorPrice
    }
  }
}
```

Reply:
This is because the association is only done when a new collection is created. Its an outlier with this deployment since I used grafting feature. But if you check the dev subgraph, you can see it working properly.